src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,25 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
```

Comment:
Looks right according to your tests but not right according to your the task.

Reply:
let mentor decide